### DISCUSSION TITLE
It is possible to combine the FPS Unlocker and Steam overlay ?

### DISCUSSION THREAD

**Discussion by B10v1** [34736384/genshin-fps-unlock#323] [Q&A]
Hi, 

I added Genshin to my Steam library as a Non Steam game to enable the steam web overlay to have my farming map and to show the FPS, but i do not have the uncapped FPS anymore

The FPS Unlocker do not recognize the Genshin Impact shortcut file if i create one, and if i start the game from the FPS Unlocker, its not starting through Steam and so the overlay is gone

How can i achieve to combine both ? It is possible ?

Thanks !

(PC/Windows)

**Comment by Fennyenn** [comment]
I literally redownloaded the game yesterday and im trying to find a solution to this same exact issue, I will report back in case I find a way

**Comment by B10v1** [comment]
I found a workaround to use a web page without the Steam overlay. I don’t think we can use the fps unlocker with the steam non-game feature currently.

So, I use the software Overwolf, which is safe to use. And within Overwolf, I use the “Browser” extension that I can toggle on/off with a keybind as if it was in-game.

So now, I can open the interactive map URL inside the game and do everything I want inside.

I hope we can use the fps unlocker in the future, but for now, this solution does the trick.

**Comment by Scarlaid** [comment]
Alternative for the map, I recommend [KongYingMap](https://yuanshen.site/docs/en/)

might work, might not 🏃‍♂️

**Comment by farsdewibs0n** [comment]
Alternatively, you can use GIMI with Adventure Map+ mod. Here are the links
[Genshin Impact Model Importer](https://github.com/SilentNightSound/GI-Model-Importer)
[Adventure Map+](https://gamebanana.com/mods/434773)
